pkg/ctl/cmdutils/scale.go

```diff
@@ -0,0 +1,91 @@
+package cmdutils
+
+import (
+	"fmt"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+)
+
+// NewScaleNodeGroupLoader will load config or use flags for 'eksctl scale nodegroup'
+func NewScaleNodeGroupLoader(cmd *Cmd, ng *api.NodeGroup, ngFilter *NodeGroupFilter) ClusterConfigLoader {
+	l := newCommonClusterConfigLoader(cmd)
+
+	l.flagsIncompatibleWithConfigFile.Insert(
+		"nodes",
+		"nodes-min",
+		"nodes-max",
+	)
+
+	validateNodeGroup := func(ng *api.NodeGroup) error {
+
+		if ng.DesiredCapacity == nil || *ng.DesiredCapacity < 0 {
+			return fmt.Errorf("number of nodes must be 0 or greater")
+		}
+
+		if ng.MaxSize != nil && *ng.MaxSize < 0 {
+			return fmt.Errorf("maximum number of nodes must be 0 or greater")
+		}
+
+		if ng.MaxSize != nil && ng.MinSize != nil && (*ng.MinSize > *ng.DesiredCapacity || *ng.MaxSize < *ng.DesiredCapacity) {
```

Comment:
I think this check can be removed for simplification since it's contained in the checks of lines 33 and 41, isn't it?

Reply:
yes, you are right. Let me do it once we confirm the above point about --name flag if you don't mind :)